cognite/client/_api/three_d.py

```diff
@@ -547,11 +547,11 @@ def delete(
         path = utils._auxiliary.interpolate_and_url_encode(self._RESOURCE_PATH, model_id, revision_id)
         utils._auxiliary.assert_type(asset_mapping, "asset_mapping", [list, ThreeDAssetMapping])
         if isinstance(asset_mapping, ThreeDAssetMapping):
-            asset_mapping = [asset_mapping]
+            asset_mapping = [ThreeDAssetMapping(asset_mapping.node_id, asset_mapping.asset_id)]
         chunks = utils._auxiliary.split_into_chunks(
             [a.dump(camel_case=True) for a in asset_mapping], self._DELETE_LIMIT
```

Comment:
[ThreeDAssetMapping(a.node_id, a.asset_id).dump(camel_case=True) for a in asset_mapping]

Reply:
See c6eb97d

Reply:
lgtm, please update changelog to add a fixed entry

Reply:
Fixed in d6732b2